tests/test_uorf.py

```diff
@@ -96,4 +96,41 @@ def test_intercistonic_distances(
 ):
     uorf = UORFCoordinates(five_utr=hbb_five_utr, uorf=region)
 
-    assert intercistronic_distance(five_sequence=hbb_five_utr_sequence, uorf=uorf) == expected
\ No newline at end of file
+    assert intercistronic_distance(five_sequence=hbb_five_utr_sequence, uorf=uorf) == expected
+
+
+@pytest.mark.parametrize(
+        "region, expected",
+        [
+            ((Region(start=16, end=67)), 16),
+            ((Region(start=302, end=407)), 302),
+            ((Region(start=510, end=576)), 510),
+        ]
+)
+def test_five_cap_to_uorf_distance(
+    hbb_five_utr: FiveUTRCoordinates,
+    region: Region,
+    expected: int,
+):
+    uorf = UORFCoordinates(five_utr=hbb_five_utr, uorf=region)
```

Comment:
I see this line several times in the tests. Perhaps it should be a new module-scoped fixture?

Reply:
Sorry, no. It takes a `region`, which has a test scope.